io/net/sriov_device_test.py.data/README.txt

```diff
@@ -7,6 +7,7 @@ hmc_username = Specify the HMC user name.
 hmc_pwd = Specify the HMC password.
 sriov_adapter = Specify the sriov adapter loc code for test.
 sriov_port: specify adapter port for test.
+sriov_roce: specify if the roce adapter VF port needs to be added and removed ex: True
```

Comment:
sriov_roce: specify  `True` if the roce adapter VF port needs to be added and removed else `False`

Reply:
@just change this rest looks good